src/templates/pt/2019/error.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base/2019/error.html" %}
+
+{% block date_published %}2019-11-04T12:00:00.000Z{% endblock %}
+{% block date_modified %}2020-08-06T00:00:00.000Z{% endblock %}
+
+{% block unknown_error %}Erro desconhecido{% endblock %}
+
+{% 
+  set localizedStatusCode = {
+    400: "Solicitação Incorreta",
+    404: "Página Não Encontrada",
+    500: "Erro interno do servidor",
+    502: "Erro temporário do servidor"
+  }
+%}
+{% macro expandedTitle(errorTitle) %}Error: {{ errorTitle }}{% endmacro %}
+{%
+  set localizedErrorMessage = {
+    "Unsupported year requested": "Ano solicitado não suportado",
+    "Unsupported chapter requested": "Capítulo solicitado não suportado",
+    "Unsupported language requested": "Idioma solicitado não suportado",
+    "Not Found": "O URL solicitado não foi encontrado no servidor. Se você inseriu o URL manualmente, verifique a ortografia e tente novamente."
+  }
+%}
+
```

Comment:
I think I made a mistake and this space shouldn't be there. Not sure why I asked for it! Can you remove again?

```suggestion
```